lib/inspec/schema/exec_json.rb

```diff
@@ -100,10 +100,10 @@ module ExecJson
           "description" => Primitives::STRING,
           "inspec_version" => Primitives::STRING,
 
-          # These are generated at runtime, and all except skip_message are guaranteed
+          # These are generated at runtime, and all except status_message are guaranteed
           "sha256" => Primitives::STRING,
           "status" => Primitives::STRING,
-          "skip_message" => Primitives::STRING, # If skipped, why
```

Comment:
Shouldn't this be retained for backwards compatibility? My understanding is that on a skip, both skip_message and status_message will be present.

Reply:
I'm taking advantage of the API being pre-1.0 to just drop it. `status_message` should be used. I'm adding `skip_message` in the reporters until 5.0 when they can be cut out.

Reply:
This isn't part of the (internal) pre-1.0 rundata API though; this is part of the published schema for the (external) output of the JSON reporter. This will impact people.

Reply:
Ah, people running JSON profiles generated by older InSpec versions? I'll put `skip_message` back in then

Reply:
I put it back but changed the comment.

Reply:
If this was re-added, would we need it to be added in `plugins.md` or left out due to deprecation?

Reply:
`plugins.md` documents the run data API and this isn't that.